src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,21 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = 'Under Construction';
```

Comment:
- [ ] Let's kindly adhere to the project requirements. Initialstate should have a state value `categories` that contain an empty array and not a string. It should be something like:

```.js
initialState: {
  categories: [ ] #empty array
}
```